src/wizard/github.jl

```diff
@@ -26,6 +26,12 @@ function github_auth(;allow_anonymous::Bool=true)
 end
 
 function obtain_token(; ins=stdin, outs=stdout, github_api=GitHub.DEFAULT_API)
+    if !isopen(ins) || (!Sys.iswindows() && !isopen(stdin))
```

Comment:
Can we put this check into the `while` loop so that if someone types `^D` during input, we can exit gracefully instead of infinite looping?

Reply:
Done!